.travis.yml

```diff
@@ -43,8 +43,12 @@ env:
     - ROS_DISTRO=indigo ADDITIONAL_DEBS="ros-hydro-opencv3" DEBUG_BASH='true' EXPECT_EXIT_CODE=100 # This should fail (trying from a wrong distro).
     - ROS_DISTRO=jade   UPSTREAM_WORKSPACE=file  ROSINSTALL_FILENAME=.ci.rosinstall  # Testing arbitrary file name without ROS_DISTRO suffix. As of 6/3/2016 this fails due to https://github.com/ros-industrial/industrial_core/pull/144#issuecomment-223186764
     - ROS_DISTRO=jade   APTKEY_STORE_SKS=hkp://ha.pool.sks-keyservers.vet  _DO_NOT_FOLD=true # Passing wrong SKS URL as a break test. Should still pass.
-    - ROS_DISTRO=jade   UPSTREAM_WORKSPACE=file ROSINSTALL_FILENAME=.i.do.not.exist EXPECT_EXIT_CODE=1    
-    - ROS_DISTRO=kinetic AFTER_SCRIPT='catkin --version'
+    - ROS_DISTRO=jade   UPSTREAM_WORKSPACE=file ROSINSTALL_FILENAME=.i.do.not.exist EXPECT_EXIT_CODE=1
+    - ROS_DISTRO=kinetic AFTER_SCRIPT='grep -q ID=ubuntu /etc/os-release && grep -q VERSION_CODENAME=xenial /etc/os-release'
+    - ROS_DISTRO=kinetic BEFORE_SCRIPT='grep -q ID=debian /etc/os-release && grep -q VERSION_ID=\"8\" /etc/os-release' EXPECT_EXIT_CODE=1
+    - ROS_DISTRO=kinetic OS_NAME=debian OS_CODE_NAME=jessie AFTER_SCRIPT='grep -q ID=debian /etc/os-release && grep -q VERSION_ID=\"8\" /etc/os-release'
+    - ROS_DISTRO=kinetic OS_NAME=debian EXPECT_EXIT_CODE=1
+    - ROS_DISTRO=kinetic OS_NAME=debian OS_CODE_NAME=xenial EXPECT_EXIT_CODE=1
```

Comment:
I see.
Then, I see [an error message on the job's output](https://travis-ci.org/ros-industrial/industrial_ci/jobs/240320321#L122):
> Tag xenial not found in repository docker.io/library/debian

I'd suggest to make the error even more user-friendly, hoping `industrial_ci` to be a convenient tool for engineers without devOps type of skills.

I guess maybe `OS_NAME=debian` is clear enough hopefully. So we might want to suggest possible options of `OS_CODE_NAME` for the corresponding `OS_NAME`?